driver/registry_sql_test.go

```diff
@@ -0,0 +1,24 @@
+package driver
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/ory/hydra/driver/config"
+	"github.com/ory/hydra/persistence/sql"
+	"github.com/ory/x/configx"
+	"github.com/ory/x/logrusx"
+)
+
+func TestDefaultKeyManager_HsmDisabled(t *testing.T) {
```

Comment:
Could you also add a test for HSM enabled?

Reply:
Yes this test was indeed missing thanks! HsmEnabled test now tests default keymanager to be hardware key manager.